packages/oauth/src/index.ts

```diff
@@ -250,30 +292,51 @@ export class InstallProvider {
         }) as unknown as OAuthV2Response;
 
         // get botId
-        const botId = await getBotId(resp.access_token, this.clientOptions);
+        const authResult = await runAuthTest(resp.access_token, this.clientOptions);
+        const botId = authResult.bot_id;
+        const orgDashboardGrantAccess = authResult.url;
```

Comment:
@stevengill Is there any reason you chose this name for the URL? As we fetch the value from `auth.test` this way:

```json
{
  "ok": true,
  "url": "https://your-domain.enterprise.slack.com/",
  "team": "Org Name",
  "user": "org-level-app",
  "team_id": "E111",
  "user_id": "W111",
  "bot_id": "B111",
  "enterprise_id": "E111",
  "is_enterprise_install": true
}
```

plus, the value is just `https://{org domain}.enterprise.slack.com/`, I prefer using a name indicating this is a URL. 

Let me share a few candidates:

* 1: `orgDashboardBaseUrl`, `org_dashboard_base_url` in Python
* 2: `enterpriseUrl` in JS/Java, `enterprise_url` in Python
* 3: `enterprise.url` in JS, `enterprise_url` in Python, `enterpriseUrl` in Java

I like 3: the most as it's easy to understand what it is.

Reply:
Kaz and I discussed this elsewhere and decided to use `enterpriseUrl` as the new name going forward.